### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by maya88en**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                        <a href="#">Terms of Use</a>.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!--END third logic section with main content : FORM-->
+
+    <!--START fourth logic section : FOOTER-->
+    <div class="footer_container">
```
Comment (bellom) [inline index.html]: footer tag should not be inside a div tag, kindly correct this.....click [here](https://www.themelocation.com/best-html5-practices/) to check the right way to do it.

**Review by bellom — CHANGES_REQUESTED** [review]
Hi @maya88en @Anna-Myzukina,
Great work guys....just few things left.
Here is the checklist for your project.
- To consider the project 'complete' it must satisfy the following points:
## General
- [x] The project has a descriptive README.md 
## Intuit logo
* [x] It shows the Intuit logo above the form
* [x] The intuit logo is an embedded image that is horizontally centered
## Intuit products
* [ ] There is a dedicated container for displaying intuit's products (mint, quickbox, turbotax)
* [ ] The products are displayed in a single row
* [ ] The products row is under the Intuit logo and above the form
* [ ] It shows the Mint logo and text (logo on the left, text on the right)
* [ ] It shows the Quickbooks logo and text (logo on the left, text on the right)
* [ ] It shows the Turbotax logo and text (logo on the left, text on the right)
## kindly check out the mint original sign up page to see the logos....
After fixing the _**unchecked**_ issues please send another request to re-review this project.

**Comment by Torres-ssf** [comment]
Hi there @maya88en @Anna-Myzukina...
Your project looks very good 👏
I noticed that you added the 3 missing logos to the HTML code but it seems that the image files are missing from your project. Before I give your project a 👍, please kindly add the 3 image files to your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                    Already have an account? 
+                    <a href="#" class="header_form_link">Sign In</a>
+                </p>
+            </div>
+            
+            <div class="form_body">
+                <form class="form_fields" action="#" method="POST" name="registration_form">
+                    <fieldset>
+                    <label for="email" class="form_label">Email adress:</label>
+                    <input id="email" class="form_input" name="email" type="text" value="" required="required" pattern="[A-Za-z0-9]" maxlength="100" title="You'll use this as your user ID">
```
Comment (Torres-ssf) [inline index.html:97]: You can use the value `email` for the input attribute `type`. It will make your code even better.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by Torres-ssf** [comment]
### Congratulation team 👍
You can submit to master now. Don't forget to submit the form.

**Pull request merged**